tests/gui/test_video_player.py

```diff
@@ -110,3 +110,109 @@ def test_QtTextWithBackground(qtbot):
 
     scene.addItem(txt)
     qtbot.addWidget(view)
+
+
+class temp_event:
+        def __init__(self, button=None, modifier=None, pos=QtCore.QPoint(5, 8)):
+            self._button = button
+            self._modifiers = modifier
+            self._pos = pos
+        
+        def button(self):
+            return self._button
+        
+        def modifiers(self):
+            return self._modifiers
+
+        def pos(self):
+            return self._pos
+
+def test_AddRemoveNodes(small_robot_mp4_vid, centered_pair_labels):
```

Comment:
Pytest gets hung up if you try to interact with any Qt classes without passing in the `qtbot` fixture. Add `qtbot` and that should allow this test to run
```suggestion
def test_AddRemoveNodes(qtbot, small_robot_mp4_vid, centered_pair_labels):
```